ui/desktop/tests/acceptance/origin-test.js

```diff
@@ -178,12 +178,15 @@ module('Acceptance | origin', function (hooks) {
   });
 
   test('can reset origin before authentication', async function (assert) {
-    assert.expect(4);
-    assert.notOk(mockIPC.origin);
+    assert.expect(3);
     await visit(urls.origin);
+    await a11yAudit();
     await fillIn('[name="host"]', window.location.origin);
     await click('[type="submit"]');
-    assert.equal(mockIPC.origin, window.location.origin);
+    assert.equal(
+      this.owner.lookup('controller:origin').origin,
```

Comment:
This seems unrelated to the PR.  I'm not seeing an `origin` controller in Desktop so I'm curious where this comes from and why the change makes sense.  Can this go in a separate PR?

Reply:
Sure. I'll break this out into a new PR.